### DISCUSSION TITLE
Additional Tags parameter is not usable

### DISCUSSION THREAD

**Discussion by JessieWang-TW** [cn-terraform/terraform-aws-networking#15] [General]
Hi,

Thank you for developing this module. Currently our team is using it but we found that addtional_tags parameter is not functioning, could you please fix that?

Kind regards

**Comment by jnonino** [comment]
Hi, I have tried to create a plan and I see the additional tags there.
I have updated [the example project](https://github.com/cn-terraform/terraform-aws-networking/blob/main/examples/test/main.tf) to use additional tags and see that it is working.
I released an updated version with those changes (2.0.16).
Can you try again and see how it works?

**Comment by jnonino** [comment]
Also, all modules are fully opened for contributions so in case your team needs to change something in the modules or thinks there is something to be fixed feel free to fill an issue and to contribute to improve the module

**Reply by JessieWang-TW** [reply]
Thank you Julian! Yes the new version 2.0.16 works fine :)
